### DISCUSSION TITLE
Gopro Hero 10,11,12 -> Selecting Microphones for audio recording

### DISCUSSION THREAD

**Discussion by Postilokero** [gopro/labs#882] [Feature Requests]
Hey ! 
Im thrilled to be able to request feature for Gopro Labs, i have been very happy to use your software for my gopros.
One feature i have always been missing, is ability to select wich microphones of gopro's 3 mics are capturing audio.
Example it would be amazing to be able to select audio capture only from front facing microphone as a mono audio track, or only to get audio from upper micophone. Also for stereo setting, example only to use front facing, and upper microphone.

If you are able to make this happen, i would be very happy :) ! Many thanks for your work already !

**Comment by dnewman-gpsw** [comment]
Some of these controls have existed for a while, no idea how well it still works. Looks for "Audio and MediaMod extensions" within https://gopro.github.io/labs/control/extensions/

**Reply by Postilokero** [reply]
Ah i see, i didnt realize it was "mute" option. Do you have any idea wich channels are wich microphones on gopro hero 10 black ?
Well im gonna test this when i get home, many thanks ! :)

**Reply by Postilokero** [reply]
Im going to reply to myself, maybe someone will also want to know channels and mics. 
So Setting 1. Front mic 2. Sidemic 3.UpperMic. Channel 4 is also upper mic, but i dont know what difference there is for channel 3.
